.github/workflows/bump-version.yml

```
@@ -0,0 +1,299 @@
+
+          # Read AI reasoning
+          REASONING=$(cat /tmp/reasoning.txt || echo "Manual version bump")
+
+          # Read key changes
+          KEY_CHANGES=$(cat /tmp/key_changes.txt || echo "")
+
+          # Generate PR body
+          cat > /tmp/pr_body.md << 'EOF'
+## Version Bump: v$CURRENT_VERSION → v$NEW_VERSION
+
+This PR bumps the version from **v$CURRENT_VERSION** to **v$NEW_VERSION** ($BUMP_TYPE_UPPER bump).
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)